bin/MadGraph5_aMCatNLO/cards/production/13TeV/VLferm_EWcouplings_4321/VLferm_EWcouplings_4321_m900GeV_EE_to_4b_customizecards.dat

```diff
@@ -0,0 +1,7 @@
+../../../../cards/production/13TeV/VLferm_EWcouplings_4321/param_card.dat
+set param_card mln 900
+set param_card mle 900
+update dependent
```

Comment:
what will be changed by this command?

Reply:
I thought that this was necessary for the correct width computations (I'm not sure why). I was mistake, I've removed this.

Reply:
I never tried this command. I would use  
set param_card decay PDGID auto 
but if you are telling me that the width is correctly updated I have no strong preference.

Reply:
The width is updated, although see the note number 2 above, there does seem to be some difficulty in in the computation of the width for the neutral VLL, which we currently correct by hand after-the-fact. But I think this is unrelated to the syntax used for this command. 

I'm happy to try your syntax if you think there is any reason for it to be preferred.

Reply:
For completeness, I also tried the other syntax you suggested, and it made no difference